resume-buddy/src/main/java/com/google/sps/data/Reviewee.java

```diff
@@ -27,7 +27,7 @@ public Reviewee(
     this.email = email;
     this.school = school;
     this.year = SchoolYear.valueOf(year.toUpperCase());
-    this.career = Career.valueOf(career.toUpperCase());
+    this.career = career.toUpperCase();
```

Comment:
You won't have to convert this string to uppercase if it can only originate from the .json file. Unless the user can specify their own string for a career that isn't listed in the .json.

Reply:
There is a text input for career in case it's not listed, but currently it is not handled in the Servlets. I'm going to add that to Issues.